example/config.sample

```diff
@@ -3,12 +3,8 @@
   "torii_port" : 50051,
   "internal_port" : 10001,
   "database": {
-     "host": "localhost",
-     "port": 5432,
-     "user": "postgres",
-     "password": "mysecretpassword",
-     "working database": "iroha_data",
-     "maintenance database": "postgres"
+     "type": "rocksdb",
+     "path": <path_to_wsv_folder>
```

Comment:
suggest use 
```suggestion
     "path": "./rocks.db",
```
relative to PWD current working directory.

```
[2021-07-27 00:32:52.377018000][E][Init]: Failed reading the configuration: File 'example/config.sample' could not be read.
```

Reply:
/path/to/wsv/folder